### ISSUE TITLE
thread failed with: fread() failed

### ISSUE THREAD

**Issue by eze21r** [madMAx43v3r/chia-plotter#446]
already got 20 plots with madmax.. suddenly stopped at phase 2-3.. I've got this error message

terminated called after throwing an instance of 'std::runtime_error'
  what():   thread failed with: fread() failed
Aborted (core dumped)

please help 🙏🏿

thank you

**Comment by madMAx43v3r** [comment]
disk failure?

**Comment by samuelflorez** [comment]
I have the same issue but it does not even allow for the start of the plotting to begin using Ram as the second temporary directory. I have about 32 GB of Ram and created a 10 GB Ram mnt/ramdisk drive and have AMD® Ryzen 7 5800x 8-core processor × 16 (16 threads available and used 10 for the madmax plotter). P.S. Thank you for your contributions

**Comment by Hardcore-fs** [comment]
> disk failure?
something else is going on......
i've had it on a couple of long runs when i tried to restart the plotter.
a restart fixes it..

**Comment by eze21r** [comment]
yes.. it's disk failure, i try to trim it and it run again nicely

thank you

**Comment by vvavepacket** [comment]
@eze21r how did you try to trim it?

Im getting that error,, do you run the trim command manually or what?

**Comment by eze21r** [comment]
@vvavepacket yeah, manually with this command

sudo fstrim -v /

**Comment by Hardcore-fs** [comment]
I keep getting this and yet the whole system checks out.
ram is ECC
ssd is raid & showing zero errors.
but a reboot fixes it every time.

**Comment by rich678s** [comment]
> I keep getting this and yet the whole system checks out.
> ram is ECC
> ssd is raid & showing zero errors.
> but a reboot fixes it every time.

same

**Comment by Hardcore-fs** [comment]
Been getting it for weeks, and i keep saying there is a "leak".
only solution i found was a reboot.

**Comment by OmarGC** [comment]
Please help!
I have the same problem but I unmount the ssd, how did you solve it?

And I used that ssd on windows with madmax and everything is fine, but on Ubuntu 20 it fails. My ssd says it is 79% health

**Comment by silverstan** [comment]
I had the similar issue with my RAID 0 disk. One disk failed. 
Removed the failed disk and made new RAID array, everything working fine now.
